cmd/release-controller-api/http.go

```
@@ -727,6 +728,246 @@ func (c *Controller) apiReleaseInfo(w http.ResponseWriter, req *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(code)
+	response := releasecontroller.APIError{
+		Code:    code,
+		Message: message,
+	}
+	data, err := json.MarshalIndent(&response, "", "  ")
+	if err != nil {
+		klog.Errorf("Failed to marshal API error response: %v", err)
+		return
+	}
+	w.Write(data)
```

golangci-lint (2.11.4)

[error] Error return value of `w.Write` is not checked

(errcheck)